api/src/main/java/org/pmiops/workbench/auth/DelegatedUserCredentials.java

```diff
@@ -88,7 +89,7 @@ public DelegatedUserCredentials(
     this.scopes = scopes;
```

Comment:
Do we need to accept a null list here? If possible I would avoid this kind of convention and just use overloaded constructors.